.github/workflows/copy-site-to-sourceforge.yml

```diff
@@ -6,17 +6,22 @@ name: "R: Copy Site to SourceForge"
 on:
   workflow_dispatch:
     inputs:
-      release-version:
+      version:
         description: 'Release Version without (-SNAPSHOT)'
         required: true
+  workflow_call:
+    inputs:
```

Comment:
Better to keep workflow_call in workflow introduction PR.

Reply:
So, close this PR then?

Reply:
Keep changes in input syntax, and let's merge

Reply:
Wait, I dont get what you mean. So just leave changes in `copy-site-to-sourceforge.yml` where `github.event.inputs.release-version` -> `inputs.version` and `release-version` -> `version`?

Reply:
Done. Hopefully that's what you meant.